### PR TITLE
[ADD] purchase_invoice_no_zero_lines: avoid zero quantity lines in ac…

### PR DESCRIPTION
**Pull request by jonerikceberio**
…count invoice lines from purchase
To answer OCA/OCB#814
While coding I found nearly the same in account-invoicing/purchase_stock_picking_return_invoicing

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I talk about `purchase_invoicing` instead of `purchase_invoice`, but I don't want to be picky...

File: purchase_invoicing_no_zero_line/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    @api.onchange('purchase_id')
+    def purchase_order_change(self):
+        """
+        after the creation of the lines, delete the zero qty lines
+        if the journal is marked as such
+        """
+        res = super(AccountInvoice, self).purchase_order_change()
+        if self.journal_id and self.journal_id.avoid_zero_lines:
+            self.invoice_line_ids -= self.invoice_line_ids.filtered(
```
Comment (pedrobaeza) [inline purchase_invoicing_no_zero_line/models/account_invoice.py:20]: Maybe for being more exact, we should filter only the lines of the added purchase order, isn't it?

File: purchase_invoicing_no_zero_line/models/account_journal.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+from odoo import api, fields, models, _
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    avoid_zero_lines = fields.Boolean(
+        string="Avoid zero lines?",
+        help="If this option is checked, the purchase invoices in this journal will have no lines with zero quantity"
```
Comment (pedrobaeza) [inline purchase_invoicing_no_zero_line/models/account_journal.py]: Please restrict to PEP8 80 cols

**Comment by pedrobaeza** [comment]
No need to copy SVG file, as it's unmodified.

**Comment by jonerikceberio** [comment]
Uups, also adding spanish translation

**Comment by jonerikceberio** [comment]
Travis hiccup, is there a way to retry?

> $ curl -sSf -o python-3.5.tar.bz2 ${archive_url}
curl: (56) SSL read: error:00000000:lib(0):func(0):reason(0), errno 104
Unable to download 3.5 archive. The archive may not exist. Please consider a different version.

**Comment by pedrobaeza** [comment]
Yeah, sometimes happens. Done.

**Comment by jonerikceberio** [comment]
@pedrobaeza , what is with codecov?

**Comment by pedrobaeza** [comment]
It's a metric for seeing test coverage. As your module doesn't have tests, then it's lowering the repository total coverage, and thus the red cross. They are not mandatory, but recommended.

File: purchase_invoicing_no_zero_line/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline purchase_invoicing_no_zero_line/__init__.py]: This is not needed in any Python file since v11 (due to Python 3)

**Review by pedrobaeza — APPROVED** [review]

**Review by enriquemartin — APPROVED** [review]

**Pull request merged**